src/analyze-video/main.cpp

```diff
@@ -6,6 +6,11 @@
 #include <opencv2/imgcodecs.hpp>
 #include <opencv2/highgui.hpp>
 
+#if  CV_MAJOR_VERSION >= 4
```

Comment:
I do not like the preprocessor way very much (preprocessor is evil, use it sparingly).

Anyway, opencv 3 has both`CV_CAP_PROP_POS_FRAME` and `cv::CAP_PROP_POS_FRAMES` I think we should just use the later everywhere instead. For example `src/fort/studio/widget/TrackingVideoPlayer.cpp` uses the later in `TrackingVideoPlayerTask::processFrame()` 

By the way, analyze video is just some tool I use to make some check on video file during development. It is not something to be released.